deploy/kubernetes/eck/es_kibana.yaml

```diff
@@ -0,0 +1,40 @@
+# This sample sets up a an Elasticsearch cluster along with a Kibana instance,
+# configured to be able to communicate with each other
+apiVersion: elasticsearch.k8s.elastic.co/v1alpha1
+kind: Elasticsearch
+metadata:
+  name: elasticsearch-sample
+  namespace: kube-system
```

Comment:
Is there a particular reason why you are deploying everything into `kube-system`? I always thought of it as owned by k8s itself.

Reply:
Hey @pebrc , sorry for the late response here! The reason for this is that wanted to have everything in the same namespace (ES, metricbeat, kube-state-metrics). So the limitation originated from the `kube-state-metrics`' namespace. In which namespace do you deploy `kube-state-metrics`?